### PR TITLE
Taxonomies

### PR DESCRIPTION
**Pull request by camdendotlol**
# Summary

This PR adds a new `Taxonomy` project model with a default `name` string field.

The purpose of taxonomies is to provide any number of controlled vocabularies that can be connected to any number of models. For example, a user might create a Genres taxonomy and connect it to a Songs model and an Albums model. This way, the user can reuse the same controlled vocabulary list anywhere it might be relevant.

Currently, this feature will likely be used for La Supplique's `keywords` the `subject` feature request by Bischoff and/or Rumpf.

### CODE REVIEW

File: app/policies/core_data_connector/taxonomy_policy.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      @current_user = current_user
+      @taxonomy = taxonomy
+
+      @project_model_id = taxonomy&.project_model_id
+      @project_id = taxonomy&.project_id
+    end
+
+    def permitted_attributes
+      [ *ownable_attributes,
+        :name,
```
Comment (dleadbetter) [inline app/policies/core_data_connector/taxonomy_policy.rb:17]: Extra comma.

**Comment by camdendotlol** [comment]
@dleadbetter It doesn't look like this repo is part of the ZenHub board. I was able to link https://github.com/performant-software/core-data-cloud/issues/48 to the PR on that repo last week, but this PR doesn't show up, nor do any of the others from this repo.

File: app/models/core_data_connector/taxonomy.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module CoreDataConnector
+  class Taxonomy < ApplicationRecord
+    include Ownable
```
[the remark this answers is not in the document]
Comment (camdendotlol) [reply app/models/core_data_connector/taxonomy.rb:3]: I've added this but I'm not sure how to test it.

[the remark this answers is not in the document]
Comment (camdendotlol) [reply app/models/core_data_connector/taxonomy.rb:3]: @dleadbetter Done. I successfully populated a Typesense index with test Taxonomy records.

**Review by dleadbetter — APPROVED** [review]

**Comment by dleadbetter** [comment]
> @dleadbetter Is this PR good to merge now or should it wait until the Sources PR is ready?

Should be good to merge 👍 .

**Pull request merged**